code/lab1-tdd-arabic-roman-converter/src/test/java/ru/unn/agile/ArabicRomanConverter/RomanToArabicTest.java

```diff
@@ -0,0 +1,128 @@
+package ru.unn.agile.ArabicRomanConverter;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RomanToArabicTest {
+
+    private int convertToArabic(final String roman) {
+        RomanToArabicConverter converter = new RomanToArabicConverter();
+        int arabic = converter.convert(roman);
```

Comment:
what will happen if you call converter.convert("BUGAGASHENKA") ? 
I guess you should add test with value that actually not a roman at all. And check it in convert before converting and throw exception if it isn't roman.

Reply:
Done